src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -370,6 +378,31 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
         }
     }
 
+    /** Reposition the placement of the overlay */
+    private repositionOverlayPlacement(): void {
```

Comment:
Can you refactor the code that has been copy/pasted from `getOverlay`, to eliminate duplication?